{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "55118aea_23da213b",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 4690
      },
      "writtenOn": "2021-10-15T21:06:02Z",
      "side": 1,
      "message": "Thanks for this! I just went through a problem like this where my LIBS_FROM_GIT was not being honored and I could not figure out how to resolve it for some oslo lib changes I am trying to test in another project. I think these additions to the error message would help a lot in the future.",
      "revId": "e5c23050ac9e2cde96e3bc476b2f784e4b3324f6",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    }
  ]
}